torch/_torch_docs.py

```diff
@@ -11423,7 +11423,7 @@ def merge_dicts(*dicts):
 add_docstr(
     torch.rot90,
     r"""
-rot90(input, k=1, dims=[0,1]) -> Tensor
+rot90(input, k=1, dims=(0, 1)) -> Tensor
```

Comment:
Why change list to a tuple here?

Reply:
The default value is a tuple in the stub file. The list is converted to tuple in torchgen.

https://github.com/pytorch/pytorch/blob/8c2c3a03fb87c3568a22362d83b00d82b9fb3db2/torchgen/api/python.py#L276-L282